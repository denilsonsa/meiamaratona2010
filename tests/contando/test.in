5 10 2
2 3 5 7 13
5 30 2
2 3 5 7 13
5 10 2
2 3 5 7 13
5 30 2
2 3 5 7 13
3 30 1
2 3 5
5 120 2
2 3 5 7 13
10 100000000 2
2 3 5 7 11 13 17 19 23 29
20 100000000 2
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 3
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 4
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 5
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 6
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 7
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 8
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 9
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 10
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 11
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 12
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 13
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 14
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 15
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 16
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 17
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 18
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 19
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
20 100000000 20
2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71
8 24171380 8
925937 105229 497269 377071 25057 999683 281719 975089
14 20413019 8
486281 3407 257189 67709 920273 250963 79193 49927 692647 317887 371057 348077 583249 921463
20 8881730 18
889703 7039 452227 333227 269473 218947 744353 324689 330623 966013 39161 405893 658991 44617 434111 405709 430579 784837 200671 264179
18 92402218 7
489959 998687 655111 348253 106619 916049 773491 528091 705113 280871 514499 460289 47777 775043 123091 936937 611101 962783
15 84810409 7
719839 691687 865313 521429 973823 867011 557159 810587 56569 184087 774703 870229 763811 340849 805891
13 97348957 13
698017 980899 306367 274259 432659 513649 328901 352819 194471 916441 915601 184837 573343
12 15365941 12
829627 971821 855863 915973 238361 780193 337907 649379 250643 177743 768073 307919
14 45044209 9
805339 256813 345733 900607 916121 154079 348463 89849 5171 944329 804743 843539 144941 861109
8 16405916 3
41017 239441 452443 725273 287173 162499 441101 109001
19 2020293 2
362377 18839 182431 144461 217223 703819 182639 442987 762407 244219 624829 228509 533129 474049 2617 919063 876971 504727 191137
15 42214146 9
939193 367273 607151 693317 656939 171827 516209 537781 564457 565979 856949 818113 575431 285667 778541
14 73277373 8
574033 276977 587827 249541 757403 698239 750713 782839 509723 36497 102359 744283 49037 904999
8 53016971 3
272927 950633 624133 468773 97847 940297 105619 918913
19 92643045 4
420253 672317 380291 411143 531197 374683 432659 262253 502643 885473 917141 952183 270553 697063 376081 623437 702353 278813 866737
20 32874965 20
32779 730679 114343 287681 292661 901183 188527 282959 830413 378661 349637 121553 502787 347143 883061 543253 519107 270131 633379 167879
5 40850675 5
155723 832379 122971 852323 18899
11 22112607 8
400067 468473 851231 67121 894233 271861 535487 383557 2447 819943 19993
9 4182436 5
255149 269879 808421 322229 40423 450101 473101 509543 644593
6 11196034 4
536453 774803 477671 546943 38273 637939
3 49439359 3
583031 695879 174901
2 20091662 1
566681 270037
15 87321963 9
498527 63949 811277 996811 490267 473503 155201 72497 496789 340237 977167 686911 522947 532313 295123
8 68485823 8
421783 324811 892559 100501 644767 559667 287107 816947
13 17049091 2
74099 15149 510061 465061 158959 974551 793669 426331 887069 358313 614417 20897 124001
14 99824281 12
566707 326567 607261 56681 33329 684527 605543 524803 807151 702439 429631 810487 776177 873641
11 82842445 10
662639 497999 640333 744137 232117 871727 81421 76651 73709 217937 405949
13 70357676 2
374789 511811 655927 49279 736399 234461 841297 20123 981913 350737 172097 748603 602279
7 75608151 6
544613 452041 540779 659327 288481 900121 263911
7 13698780 5
966781 905959 691051 387839 158621 503939 46327
6 5750564 5
671039 213097 309521 668903 893479 241823
4 29349882 3
556823 729719 77339 637937
20 23644704 18
728681 892169 938869 985213 851117 397799 576341 711629 332273 46723 815273 34039 641441 316853 429109 18521 596261 169063 916109 631487
17 33302731 7
927869 806893 403603 431237 392467 240287 174703 316957 201337 190823 322409 338497 925943 322243 766333 66179 734803
11 31161932 1
704233 419513 785249 690721 259033 412487 70139 859603 868697 723089 241981
9 79214144 4
498923 600043 285317 320609 550189 803461 512821 633937 589139
15 83459685 7
817891 361651 12479 244003 210857 325723 505447 111091 787547 653357 5939 569021 434647 631061 749893
8 23902157 5
860747 630713 979987 393583 625283 786889 65003 118081
4 70146514 4
193859 488627 26347 815819
17 8664253 7
501821 189733 422761 448207 751307 794921 726839 48527 961099 582167 752627 536147 843881 504683 141653 325993 103423
10 19057921 8
591443 180071 521693 279187 948293 412603 818659 745531 691337 31991
11 36272179 10
107693 221327 370247 641441 330607 161047 491353 126127 842291 671161 739369
7 49413376 4
680707 124783 18433 642241 621779 355007 402613
7 10659715 7
810949 844463 776987 90803 493201 247439 249127
2 14900732 2
807473 889349
14 68057656 8
982351 492979 451747 251857 412277 14831 317437 496193 501817 853837 50909 683983 414809 15077
8 43283321 8
855511 632627 619117 565637 441887 398989 223463 107101
17 1413536 4
171467 115361 18397 760457 745903 468737 870731 281579 523403 507119 687917 329663 182443 428569 264779 201287 502421
11 41373837 9
184913 307451 975739 231299 740969 768139 729041 507371 530701 254489 616219
15 45922632 4
797333 229547 935531 227131 520691 877073 913433 214003 739511 394063 787091 968467 411409 646189 657959
2 53247780 1
791969 798079
0 0 0
