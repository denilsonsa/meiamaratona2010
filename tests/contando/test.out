2
11
2
11
14
38
30948713
31606529
16637670
5476476
1164195
188594
25171
577
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
109
0
0
0
0
0
0
0
0
0
0
0
0
1083
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
133
